### DISCUSSION TITLE
Word/Character count for a selection

### DISCUSSION THREAD

**Discussion by arielsvg** [standardnotes/forum#2611] [Suggestions] [answered]
It would be great if you could get the word/char count of a selection as well!

_Originally posted by @plantfansam in https://github.com/standardnotes/web/issues/193#issuecomment-583477293_

**Comment by arielsvg** [comment]
Currently the action bar shows stats like word/char count for the whole note; showing stats for a selection would be pretty nifty too!

**Comment by myreli** [comment, marked as answer]
Closing to track on https://github.com/standardnotes/forum/discussions/2675.
